app/views/mountain_view/_styleguide_intro.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="mv-main__header">
+  <h1>Styleguide</h1>
+</div>
+<div class="mv-main__description">
+  Select one of the components from the side to view its examples and documentation.
+  <br />
+  <br />
+  To replace the content on this page, add a folder mountain_view in your views and create a partial
```

Comment:
Would it be mountain_view or your style guide location in these instructions?

Reply:
This would be `mountain_view`, yes.

Couple suggestions:
- folder => directory
- How about making it more explicit, showing the path to the views directory, eg:
```suggestion
  To replace the content on this page add the directory `app/views/mountain_view` in your application, and create a partial named
```